dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterableTest.java

```diff
@@ -44,6 +45,9 @@ public class ZonedIntervalIterableTest {
   @SuppressWarnings("UnusedVariable")
   private static final Logger LOG = LoggerFactory.getLogger(ZonedIntervalIterableTest.class);
 
+  private static final TimeTruncator TRUNCATOR =
+      TimeTruncator.basedOnChronoUnit(ChronoUnit.SECONDS);
```

Comment:
Optional: Would it make sense to have different tests use different truncation unit, e.g. keep most of the tests truncating to minutes plus have one test with seconds and one with hours?

Reply:
I feel having test for `TimeTruncator` using different `ChronoUnit` values should be enough for now.